scripts/buyV1Covers.js

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
```

Comment:
`scripts/buyV{1,2}Covers.js` seem to be testnet/local scripts, not something we would use on mainnet. Is it possible to move them to a separate folder that would have a name indicating their purpose?

Reply:
Yeah, they are only for testing purposes. I'll move them to a separate folder 👍